app/src/main/java/com/bogdan/codeforceswatcher/features/contests/ContestsFragment.kt

```diff
@@ -41,7 +41,7 @@ class ContestsFragment : Fragment(), SwipeRefreshLayout.OnRefreshListener,
 
     override fun onRefresh() {
         store.dispatch(ContestsRequests.FetchContests(isInitiatedByUser = true))
-        Analytics.logContestsListRefresh()
```

Comment:
Haha, the idea was to track user's interest in concrete data, not `swipe-to-refresh`. Revert it, please.

Reply:
Well, you are doing it in another place, but it's not obvious at all, when reading this method call. You can pass a key or better enum.